game-frontend/src/components/user-script/index.tsx

```diff
@@ -29,7 +31,8 @@ const UserScript: FC = () => {
   return (
     <Container data-testid="user-script-form">
       <form>
-        <Textarea onChange={onChange} rows={8}></Textarea>
```

Comment:
Make sure to update the userScript - atm it's not synced and therefore the code written is lost.

Reply:
will do, thx